### PR TITLE
Update integrate-configuration-manager-with-mdt.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #4497

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/deploy-windows-mdt/integrate-configuration-manager-with-mdt.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ MDT is a free, supported download from Microsoft that adds approximately 280 enh
 
 As noted above, MDT adds many enhancements to Configuration Manager. While these enhancements are called Zero Touch, that name does not reflect how deployment is conducted. The following sections provide a few samples of the 280 enhancements that MDT adds to Configuration Manager.
 
+>[!Note]
```
Comment (nenonix) [inline windows/deployment/deploy-windows-mdt/integrate-configuration-manager-with-mdt.md]:
```suggestion
> [!Note]
```

File: windows/deployment/deploy-windows-mdt/integrate-configuration-manager-with-mdt.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ MDT is a free, supported download from Microsoft that adds approximately 280 enh
...
+>Microsoft Deployment Toolkit requires [Windows PowerShell 2.0 Engine](https://docs.microsoft.com/powershell/scripting/install/installing-the-windows-powershell-2.0-engine) to be installed on the server.
```
Comment (nenonix) [inline windows/deployment/deploy-windows-mdt/integrate-configuration-manager-with-mdt.md]:
```suggestion
>Microsoft Deployment Toolkit requires you to install [Windows PowerShell 2.0 Engine](https://docs.microsoft.com/powershell/scripting/install/installing-the-windows-powershell-2.0-engine) on your server.
```

File: windows/deployment/deploy-windows-mdt/integrate-configuration-manager-with-mdt.md [hunk 3]
```diff
@@ -27,6 +27,10 @@ MDT is a free, supported download from Microsoft that adds approximately 280 enh
...
+
```
Comment (nenonix) [inline windows/deployment/deploy-windows-mdt/integrate-configuration-manager-with-mdt.md]:
```suggestion
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix ,
your recommendations were applied,
thank you!

**Comment by mypil** [comment]
@greg-lindsay - PR has been copy edited and is ready for final review. This is based on issue #4497 

Thank you.

**Comment by greg-lindsay** [comment]
Just FYI, this entire doc set is being rewritten right now and should be published with new information this month.

**Pull request merged**